livre/Eq/Config.cpp

```diff
@@ -40,6 +40,12 @@
 #  include <zeq/hbp/hbp.h>
 #endif
 
+#ifdef LIVRE_USE_RESTCONNECTOR
+#  include <restconnector/RestConnector.h>
+static const std::string PUBLISHER_SCHEMA_SUFFIX = "resp://";
+static const std::string SUBSCRIBER_SCHEMA_SUFFIX = "cmd://";
```

Comment:
In our current architecture we use 2 channels for communication. One for commands  ( "cmd://") and the other for responses ( "resp://" ).
I think changing that is really beyond the scope of this commit... This can change the architecture and will change RESTConnector so I think this should be discussed in more details.
I created a story for that: BVWS-229